megamek/src/megamek/client/ui/swing/EquipChoicePanel.java

```diff
@@ -1739,10 +1739,6 @@ private void refreshC3() {
                 if (entity.hasNavalC3() != e.hasNavalC3()) {
                     continue;
                 }
-                // likewise can't connect c3 to nova
```

Comment:
Removing this check lets a unit with a C3 slave but no Nova CEWS connect to a Nova network.

Reply:
But isn't it only applies on the lobby? As I said above you can't link Nova on the lobby for now. Although I am not used to the structure of MegaMek but it seems that the part is only concern about config unit on the lobby but not the other parts.

It would be possible to only modify the sentence rather than remove it, though. Someone may make the option to start with Nova links later.

Reply:
Also since it is the list of C3 links, modify it will allow a unit with Nova(but not C3) to link with an another unit with C3.

Reply:
Well, this section(config unit on the lobby) have no ability to link Nova network at all, so the phrase is unnecessary anyways. For now you cannot link Nova network because this section provides nothing for link Nova networks. It only provides the links for C3, C3i, and Naval C3.

Else, is it better to make a new request that allows to set the Nova network BETWEEN THE ANOTHER UNIT WITH NOVA on the lobby?